integrations/pinecone/pydoc/config.yml

```diff
@@ -4,9 +4,7 @@ loaders:
     modules:
       [
         "haystack_integrations.components.retrievers.pinecone.embedding_retriever",
-        "haystack_integrations.document_stores.pinecone.document_store",
-        "haystack_integrations.document_stores.pinecone.errors",
-        "haystack_integrations.document_stores.pinecone.filters",
```

Comment:
- errors: this file has been removed
- filters: this module contains the filtering logic for Pinecone. Should not be user-facing.